aioesphomeapi/connection.py

```diff
@@ -172,6 +174,8 @@ def __init__(
         self._connect_task: Optional[asyncio.Task[None]] = None
         self._fatal_exception: Optional[Exception] = None
         self._expected_disconnect = False
+        self._send_pending_ping = False
+        self._loop = asyncio.get_event_loop()
```

Comment:
We can change this to `get_running_loop()` once we drop support for older python